Buildings/Air/Systems/SingleZone/VAV/Examples/OptimalStart/BaseClasses/package.mo

```diff
@@ -0,0 +1,10 @@
+within Buildings.Air.Systems.SingleZone.VAV.Examples.OptimalStart;
+package BaseClasses "Base classes of models in the package OptimalStart"
+extends Modelica.Icons.BasesPackage;
+
+  annotation (preferredView="info", Documentation(info="<html>
+<p>
+This package contains base classes used by the example models of OptimalStart.
```

Comment:
Add the full link in the form ``<a href=\"modelica://Buildings....``. Then a hyperlink will be added, and if we rename the class, it will get updated too.